ignite/services/scaffolder/init.go

```diff
@@ -66,18 +67,18 @@ func generate(
 	absRoot string,
 	noDefaultModule bool,
 ) error {
-	gu, err := giturl.Parse(pathInfo.RawPath)
-	if err != nil {
-		return err
+	githubPath := gomodulepath.ExtractAppPath(pathInfo.RawPath)
```

Comment:
We're not limited to Github, all Git hosts should work. Can we use a general name here and similar places?

Reply:
Only here, the name is like that to be explicit because that path is only used in a readme template, is the suffix path used for the "get.ignite.com" installer in `ignite/templates/app/stargate/readme.md.plush` that only works with GitHub links as I recall.

Also have in mind that "username/" is prefixed to it for the cases where the Go module path is just a name, so the result in the readme template would be:
```
url https://get.ignite.com/username/name@latest! | sudo bash
```